### ISSUE TITLE
Please clarify the license of code vs generated shaders?

### ISSUE THREAD

**Issue by haasn** [bjin/mpv-prescalers#41]
I noticed that in your `ravu` branch, you have the `LICENSE` file set up for a BSD-3 clause license, but in your `master` branch, it instead refers to the LGPLv3 (as do your generated shaders).

Is this mismatch intentional? Do I understand it correctly that you license specifically your trained weights under LGPLv3 but the generating code under BSD3?

**Comment by bjin** [comment]
> Do I understand it correctly that you license specifically your trained weights under LGPLv3 but the generating code under BSD3?

Yes. The code from 'ravu' branch is intentionally released with a permissive license. For the sake of convenience, the generated weights was also released under LGPLv3, same with ravu shaders.

Unlike nnedi3 shaders, I'm open to release both ravu trained weights and ravu shaders under a more permissible license.

**Comment by haasn** [comment]
It would probably be useful, yes. I'm not sure how LGPLv3 interacts with the GPLv2 and the Apache-2, a permissive license would avoid all headaches (for use in VLC).
